doc/source/walkthrough.rst

```diff
@@ -292,9 +292,9 @@ Cancelling tasks
 
       from ray.exceptions import TaskCancelledError
 
-      try:
+      try: 
```

Comment:
Remove the space at the end. I think lint will fail with them

Reply:
Thanks for that catch!